apps/dialog/src/routes/-components/AddFunds.tsx

```diff
@@ -19,22 +19,26 @@ import {
   createConfig,
   useAccount,
   useAccountEffect,
+  useBalance,
   useConnect,
   useDisconnect,
   useSendCalls,
   useWaitForCallsStatus,
   useWatchBlockNumber,
   WagmiProvider,
 } from 'wagmi'
+import { CheckBalance } from '~/components/CheckBalance'
 import { porto } from '~/lib/Porto'
 import * as Tokens from '~/lib/Tokens'
 import { Layout } from '~/routes/-components/Layout'
+import { PriceFormatter, ValueFormatter } from '~/utils'
 import LucideCopy from '~icons/lucide/copy'
 import LucideCopyCheck from '~icons/lucide/copy-check'
 import LucideOctagonAlert from '~icons/lucide/octagon-alert'
 import TriangleAlertIcon from '~icons/lucide/triangle-alert'
+import Star from '~icons/ph/star-four-bold'
 
-const presetAmounts = ['30', '50', '100', '250'] as const
+const PRESET_AMOUNTS = ['30', '50', '100', '250'] as const
```

Comment:
we generally keep constants camelCase (can rename `MAX_AMOUNT` here too)

```suggestion
const presetAmounts = ['30', '50', '100', '250'] as const
```